src/argus/auth/models.py

```diff
@@ -1,5 +1,19 @@
 from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from phonenumber_field.modelfields import PhoneNumberField
 
 
 class User(AbstractUser):
     pass
+
+
+class PhoneNumber(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    phone_number = PhoneNumberField()
+
+    def __repr__(self):
+        return f"<PhoneNumber: {self.user.username}, {self.pk}>"
+
+    def __str__(self):
+        return f"PhoneNumber id {self.pk} for {self.user.username}"
```

Comment:
A much more natural string representation of a PhoneNumber object for me, would be this:
```suggestion
        return f"PhoneNumber {self.phone_number} for {self.user}"
```
`__repr__()` should include the `pk` to be unambiguous, but I think it's a lot more useful to be able to see the actual phone number in `__str__()`.

What do you think?

Reply:
My GDPR-sense doesn't like this, but we should probably go through everything with a fine-toothed GDPR-comb when everything is finished anyway. For reference, if __str__ is not set, the autogenerated __repr__ is ised, which looks like `f"<PhoneNumber object ({self.pk})>"`

Reply:
Yeah, that sounds like a good plan!